symfony.lock

```diff
@@ -898,9 +895,6 @@
             "project-base/config/packages/security.yaml"
         ]
     },
-    "symfony/service-contracts": {
-        "version": "v1.1.8"
-    },
     "symfony/stopwatch": {
         "version": "v3.4.32"
     },
```

Comment:
are you really sure with updating symfony.lock file? I remember in a different PR `symfony/service-contracts` was added

Reply:
Maybe that is mistake :D